docs/installation-and-deployment.md

```diff
@@ -365,3 +365,70 @@ command:
+
+- Customize the realm configuration
+    
+    Update the realm configuration file (`charts/keycloak/realms/realm_m4i_with_provider.json`) by entering
+    your credentials:
+    
+| Key                                            | Description                             |
+| ---------------------------------------------- | --------------------------------------- |
+| `{{ .identityProviders.config.clientId }}`     | The client ID of OAuth 2.0 client.      |
+| `{{ .identityProviders.config.clientSecret }}` | The client secret of  OAuth 2.0 client. |
+
+!!! note
+    In addition to Azure Active Directory, social login is also supported through OAuth 2.0 clients, including
+    [Google](https://keycloakthemes.com/blog/how-to-setup-sign-in-with-google-using-keycloak),
+    [GitHub](https://medium.com/keycloak/github-as-identity-provider-in-keyclaok-dca95a9d80ca) or
+    [Facebook](https://medium.com/@didelotkev/facebook-as-identity-provider-in-keycloak-cf298b47cb84).
+    For a complete list of supported identity providers, refer to the [keycloak website](https://www.keycloak.org/).
+    These applications can be configured as identity providers in Keycloak.
+
+!!! tip
+    If the deployment is already running, you can enable the identity provider directly through the Keycloak UI.
+
+**To enable active directory login on the keycloak UI:**
+
+- Navigate to the Keycloak administration console.
+- Click "Identity providers" in the menu, then choose the desired provider from the dropdown menu.
+- Set the Client ID and Client Secret. The rest of the settings can remain default.
+
+### Managing Users
+
+In **Aurelius Atlas**, you have the flexibility to control user access through role-based permissions.
+Users can either have **read-only** access or **read/edit** permissions, depending on their role within
+the system. By default, all users are assigned the **viewing** role, but administrators
+can manage and modify roles as needed.
+
+- **Viewing Role (Default):** Users with this role can **view** data but cannot make any changes.
+- **Editing Role:** Users with this role have permission to **edit** data and perform updates.
+
+### Register New Users
+
+When **active directory login** is enabled, users can easily register through the **login UI**.
+
+For greater control over user management, administrators can also **manually create users** directly
+within the **Keycloak UI**. For more information please visit the
+official [Keycloak documentation](https://www.keycloak.org/docs/latest/server_admin/#assembly-managing-users_server_administration_guide).
+
+### Managing Roles in Keycloak UI
+
+Administrators can fully manage user roles through the **Keycloak Admin UI**.
+
+**Editing User Roles:**
+
+1. In the Keycloak Admin Console, navigate to the **Users** section.
+2. Select the desired user and go to the **Role Mappings** tab.
+3. Assign `ROLE_ADMIN` to grant the user editing rights, or remove roles to make it a read-only user.
```

Comment:
If I remember correctly, you only need the data steward role to be able to edit stuff. Again, it would be good to have a matrix of all the roles and associated permission.s